easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-foss-2022b.eb

```diff
@@ -0,0 +1,46 @@
+# A. Domingo (Vrije Universiteit Brussel)
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'Simple-DFTD3'
+version = '0.7.0'
+
+homepage = 'https://dftd3.readthedocs.io'
+description = """Reimplementation of the D3 dispersion correction. The s-dftd3 project aims to
+provide a user-friendly and uniform interface to the D3 dispersion model and
+for the calculation of DFT-D3 dispersion corrections."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+toolchainopts = {'usempi': False, 'openmp': True, 'pic': True}
+
+github_account = 'dftd3'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['19400a176eb4dcee7b89181a5a5f0033fe6b05c52821e54896a98448761d003a']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+]
+
+dependencies = [
+    ('mctc-lib', '0.3.1'),
+    ('mstore', '0.2.0'),
+    # ('TOML-Fortran', '0.4.1'), # Vesion used in code is 0.2.4
+]
+
+configopts = '-DWITH_BLAS=TRUE -DWITH_OpenMP=TRUE -DBUILD_SHARED_LIBS=ON'
```

Comment:
It's a bit better to use
```
build_shared_libs = True
```
Instead of using -DBUILD_SHARED_LIBS

Reply:
Done. I was not sure which one is better to use. Thanks.